apps/fluent-tester/src/TestComponents/Separator/SeparatorTest.mobile.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
```

Comment:
What's the reason we want to separate out the test page to be mobile specific? If possible, could we move the test page to the same for and have a "mobile only" section? 

Reasoning: Less fracturing might mean it's easier to spot mobile vs desktop differences. Happy to see if I'm wrong and there's another way though.

Reply:
I've personally been pushing for test sections to be separated per platform in their implementations differ (or one platform doesn't support that feature, or something) to avoid a bunch of if (Platform) checks inside the test pages itself. I feel like it makes the test pages messy to have those checks and it also means a bunch of code that isn't used is included in the bundle, although that's less important in the tester.

Probably worth getting on the same page about how to handle these differences as part of @ankraj12 's plans

Reply:
For readability purpose - I am inclined towards separate files if there are a messy amount of "if platform " checks inline. If it's just a couple of clean forks, it could live in same file I guess. I would take a note of this in documentation.

Reply:
So for now , we're going it platform level forks for test app ?

Reply:
I'll go with what the rest choose. So please don't block on me and choose what feels best.

Reply:
Let's have a internal discussion on the doc Ankita is preparing and have a common decision and document it for everyone to refer.